docs/config_objects.rst

```diff
@@ -134,9 +134,13 @@ Galaxy Types
 
     * ``index`` = *int_value* (default = 'Sequence' from 0 to ``cosmos_catalog.nobjects``-1)  Which item in the catalog to use. Special: If ``index`` is either a 'Sequence' or 'Random' and ``last`` or ``max`` (respectively) is not specified, then it is automatically set to ``real_catalog.nobjects-1``.
     * ``gal_type`` = *str_vale* (required, unless ``real_catalog.use_real`` is False, in which case 'parametric') Which type of galaxy to use.  Options are 'real' or 'parametric'.
+    * ``noise_pad_size`` = *float_val* (default = 5) The size of a padding region around the HST image when ``gal_type='real'``.
     * ``deep`` = *bool_value* (default = False) Whether the flux and size should be rescaled to approximate a galaxy catalog with a limiting mag of 25 in F814W, rather than 23.5.
     * ``noise_pad_size`` = *float* (optional) If provided, then the original image is padded to a larger image of this size using the noise profile of the original image.  This is important if you are using ``noise.whiten`` or ``noise.symmetrize``.  You want to make sure the image has the original noise all the way to the edge of the postage stamp.  Otherwise, the edges will have the wrong noise profile.
     * ``sersic_prec`` = *float_value* (default = 0.05) The desired precision on the Sersic index n in parametric galaxies.  GalSim is significantly faster if it gets a smallish number of Sersic values, so it can cache some of the calculations and use them again the next time it gets a galaxy with the same index.  If ``sersic_prec`` is 0.0, then use the exact value of index n from the catalog.  But if it is >0, then round the index to that precision.
+    * ``chromatic`` = *bool_value* (default = False) Whether to build chromatic profiles. (Only valid if ``gal_type='parametric'``.)
+    * ``area`` = *float_value* (default = None, which will use the HST collecting area.)  The effective collecting area in cm**2 of the telescope being simulated.  Used for rescaling the flux values.
+    * ``exptime`` = *float_value* (default = 1)  The exposure time in seconds.  Used for rescaling the flux values.
```

Comment:
Somebody might wonder why is the default value "1" and not the length of the COSMOS HST exposures.

Reply:
I don't know the history of this decision.  Any insight you want to share for me to write here?

Reply:
I had to go remind myself, and my answer is not really satisfying.

It seems that somebody in the COSMOS survey decided to normalize their science release images to have units of counts/s instead of counts.  So, to get the number of counts that would've been seen in COSMOS, you effectively have to pretend they had a 1s exposure time.  I don't know why they decided to do that, but we got stuff from them that was normalized this way, so ...

Reply:
For this, the update has
```
(Note: The native COSMOS galaxy images being used have been rescaled to 1 second exposures, so the default corresponds to a 1 second exposure on HST.)
```
I guess rescaled to 1 second exposures is equivalent to rescaling to counts per second rate.  Not sure if there is any distinction worth making there.

Reply:
Hmm, I think there are some subtleties we might want to mention for people who are trying to actually match COSMOS-like images.  I checked and I think there is language about this in scene.py and real.py that we could use.  For example, in scene.py it says "The nominal COSMOS zeropoint for single-orbit depth (2000s of usable exposure time, across 4 dithered exposures) is supposedly 25.94.  But the science images that we are using were normalized to count rate, not counts, meaning that an object with mag=25.94 has a count rate of 1 photon/sec, not 1 photon total.  Since we've declared our flux normalization for the outputs to be appropriate for a 1s exposure, we use this zeropoint directly."  Knowing that the science images were normalized to count rates is important because otherwise they might think that matching the quoted zero points requires putting in 2000?  Or maybe there is simply a tweak on the sentence you included that could express this?

Reply:
I don't think the user has easy access to the zeropoint directly.  The zeropoint is used to set the flux of the returned object (right after where this comment is in scene.py), so I think it's already correctly applied.  If they want the objects to have the right flux for a different exposure time than 1 second, then I'm pretty sure using that exptime here does the right thing.  I guess I'm probably not grokking what you're worried about the user doing wrong here exactly.

Reply:
Mmm.  I might be confusing myself, so I'm not sure there is a problem.  Let me try again, and ignore the zero point distraction.  Imagine we have a user who wants to reproduce COSMOS images.  So they look up COSMOS and find that the total exposure time over all images was 2000 s.   Should they be using exptime=1 or 2000 at this point?  And how should the result compare to what is seen in the science images?

Reply:
It depends on if they want to reproduce the real COSMOS images that had effective exposure time of 2000s or if they want to reproduce the images in the file we make available.  In the former case, they should use exptime=2000.  In the latter either leave it out or explicitly set exptime=1.

Reply:
Or maybe your point is that the "real" COSMOS images (i.e. the processed science images) are the ones that were rescaled to count rate, so after processing, they have an effective exposure time of 1 s.  In that case, using exptime=1 would match the flux level.

Reply:
How about this as possibly clearer and getting at this subtlety:
```
(Note: The processed COSMOS ACS/HST science images are given in counts/second, so they have an effective exposure time of 1 second in terms of their flux levels.  The default value of corresponds to a 1 second exposure 
on HST, which will match these processed images.)
```

Reply:
I pushed an edit with this language, but please feel free to tweak further if you want.

Reply:
Thanks for parsing my wandering remarks so effectively.  I understood all this very deeply 6+ years ago but it's been far too long since I thought about it carefully!  I think your updated version is good.